06-auto-scaling/README.md

```diff
@@ -409,6 +467,8 @@ an instance is added.
 
 _After you stop consuming CPU, how long does it take now before scale-in?_
 
+> About 15 minutes
```

Comment:
Should be 5 (basic monitoring) or 1 (detailed monitoring) minutes.  Curious why it would have taken that long.